src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.2.0
+
+### New Feature
+
+- Support for self-signed and private CA with Docker
+- Buildkit port is now configurable (default 443)
+- Add option for sharing namespace with users within organization
+- UI pagination support for GitHub branches list
+- Add permanently user deletion command
+- Allow to scale down to zero nodes if no user pods are running
+- Populate oauth errors to the user
+- Added new helm values to set PSP and AppArmor
+- Update okteto cli version to 2.9.0
+
+### Bug Fix
+
+- Don't show warning on k8s 1.23.x
+- Frontend nginx CVE-2022-40303/CVE-2022-40304
```

Comment:
We should add a link to this

Reply:
@maroshii You mean a link to the CVE? like this one https://ubuntu.com/security/CVE-2022-40303?

Reply:
yes